modules/dnn/src/op_inf_engine.cpp

```diff
@@ -609,7 +609,7 @@ void InfEngineBackendNet::initPlugin(InferenceEngine::CNNNetwork& net)
             {
                 candidates.push_back(param_pluginPath);
             }
-
+#if INF_ENGINE_VER_MAJOR_LE(INF_ENGINE_RELEASE_2019R3)
```

Comment:
Despite default cpu_extension will be removed in the future version, there is still an approach to let user enable custom layers with it.